app/controllers/reviews_controller.rb

```diff
@@ -8,6 +8,7 @@ def create
     @fruit = Fruit.find(params[:fruit_id])
     @review = Review.new(review_params)
     @review.fruit = @fruit
+    @review.user_id = current_user.id
```

Comment:
you can also just do `@review.user = current_user` thanks to Active Record

Reply:
change made